www/src/pages/editor/editDedupSite/EditDedupMineralSite.tsx

```diff
@@ -143,9 +144,123 @@ export const EditDedupMineralSite = observer(({ dedupSite, commodity }: EditDedu
       message.success("Ungrouping was successful!");
     }
   };
-
+  let dynamicColumns = useMemo(() => {
```

Comment:
it's okay to move dynamic cols into the useMemo function that creates columns (merged two useMemo())